client/cli/src/main.rs

```diff
@@ -148,6 +154,9 @@ async fn main() {
             let id = client.reissue(coins, &mut rng).await.unwrap();
             info!(%id, "Started reissuance, please fetch the result later");
         }
+        Command::Validate { coins } => {
+            client.validate_tokens(coins).await.expect("Invalid tokens");
```

Comment:
I'm not sure if an `expect` is the right thing to do here

Reply:
I think so, let's see if @dpc has a different view in #416